src/profile-logic/marker-data.js

```diff
@@ -27,13 +48,14 @@ export function getJankInstances(
       data: null,
     });
 
-  let lastResponsiveness = 0;
-  let lastTimestamp = 0;
+  let lastResponsiveness: number = 0;
+  let lastTimestamp: number = 0;
   const jankInstances = [];
   for (let i = 0; i < samples.length; i++) {
     const currentResponsiveness = samples.responsiveness[i];
     if (currentResponsiveness === null || currentResponsiveness === undefined) {
-      return [];
+      // Ignore anything that's not numeric.
```

Comment:
maybe comment a bit more about why this can happen: feature isn't enabled, or the profiler skipped a value. Also I'd be very curious about why this can happen, so maybe we should file a bugzilla bug for the gecko profiler and put the bug # in the comment here?